Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -22,30 +24,22 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// </summary>
     public partial class GridSplitter
     {
+        // Symbols for GripperBar in Segoe MDL2 Assets
+        private const string GripperBarVertical = "\xE784";
```

Comment:
If the control allows for content then it should be a ContentControl. I would suggest changing to  a ContentControl, mark Element as deprecated and when it changes set the content to be the value. Default the content to be the style proposed